File: project/python_PLD/bmi.py
#!/usr/bin/python3

"""A function that calculates the bmi of the individual and checks if they are overweight, underweight or obesed or normal"""

def bmi_calculator(name, weight_kg, height_m):
    bmi = weight_kg / height_m ** 2
    
    if bmi < 18.5:
        print(f"{name} your bmi is {bmi}, and you are underweight")
    elif 24.5 <= bmi <=24.5:
        print(f"{name} your bmi is {bmi}, and you are overweight")
    elif bmi > 29.5:
        print(f"{name} your bmi is {bmi}, and you are obesed")
    elif 18.5 <= bmi <= 24.5:
        print(f"{name} your bmi is {bmi}, and you are normal")
    
    return bmi
        
bmi_calculator(name="emmanuel", weight_kg=150, height_m=17.5)
bmi_calculator("David", 200, 20)

File: project/c/c++PLD/bmi.c
#include <stdio.h>

/*Programme calculates the bmi of an individual and categorizes them*/
int main(void)
{
	float weight, height;
	float bmi;
	/*Accepting weight in kg and height in m*/
	printf("Weight in kg:\n");
	scanf("%f", &weight);
	printf("Height in m\n");
	scanf("%f", &height);
	bmi = weight / (height * height);
	if (bmi <= 18.5)
		printf("your bmi is %.2f and you are underweight\n", bmi);
	else if (bmi >= 18.5 && bmi <= 25.0)
		printf("your bmi is %.2f and you are normal\n", bmi);
	else if (bmi >= 25.0 && bmi <= 29.9)
		printf("your bmi is %.2f and you are overweight\n", bmi);
	else if (bmi >= 30.0)
	{
		if (bmi >= 30.0 && bmi <= 34.9)
			printf("your bmi is %.2f and you belong to class I obesity group\n", bmi);
		else if (bmi >= 35.0)
			printf("your bmi is %.2f and you belong to class II obesity group\n", bmi);
		else
			printf("your bmi is %.2f and you belong to class III obesity group\n", bmi);
	}
	return (0);
}